pkcs11helpers/helpers.go

```diff
@@ -54,6 +55,150 @@ func Initialize(module string, slot uint, pin string) (*Session, error) {
 	return &Session{ctx, session}, nil
 }
 
+// from src/pkg/crypto/x509/x509.go
+var (
+	oidNamedCurveP256 = asn1.ObjectIdentifier{1, 2, 840, 10045, 3, 1, 7}
+	oidNamedCurveP384 = asn1.ObjectIdentifier{1, 3, 132, 0, 34}
+)
+
+var curveOIDs = map[string]asn1.ObjectIdentifier{
+	"P-256": oidNamedCurveP256,
+	"P-384": oidNamedCurveP384,
+}
```

Comment:
I don't see any references to these vars aside from in this map, so maybe just inline them directly into the map?